src/gitgraph.js

```diff
@@ -363,13 +396,32 @@
       } else {
         commit.isMouseover = false;
       }
-    }
+    });
 
     if ( isOut ) {
       self.tooltip.style.display = "none";
     }
   };
 
+
+  /**
+   * Click event on commit dot
+   *
+   * @param {MouseEvent} event - Mouse event
+   *
+   * @self Gitgraph
+   **/
+  GitGraph.prototype.click = function ( event ) {
+    var self = this.gitgraph;
+    self.applyCommits(event, function(commit, isOverCommit) {
+      if (isOverCommit) {
```

Comment:
@terhechte > I tend to prefer writing 

```
if(!isOverCommit) return;

if (!(commit.onClick === null)) {
  commit.onClick(commit, true);
}
```

I found that's a bit more readable (less nesting).